### PR TITLE
Try out experience

### PR DESCRIPTION
**Pull request by cbordier**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Add explanation for Try out API.

## Deploy preview link

http://pr-563.amplify-central.opendocs-builder.pcloud.axway.int/docs/manage_marketplace/consumer_experience/api_tryoput/

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | 4a2e053 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-563.amplify-central.opendocs-builder.pcloud.axway.int |

**Comment by lbadenhop** [comment]
@cbordier, I reviewed the content that is currently in this PR. You will have to publish this when I am on vacation.

**Comment by cbordier** [comment]
Thank you @lbadenhop

File: content/en/docs/manage_marketplace/consumer_experience/api_tryoput.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+Once pasted, click **Authorize**. Depending on the flow, you may be redirected to the authentication server to authenticate the client ID and client secret and then get back to the Marketplace where all values change to stars, and **Authorize** changes to **Logout**. If you logout, the API Key is erased, and you must start the process again. If you click **Close**, the side panel is closed, and you are now ready to [try a method](#try-a-method).
+
+### Bearer authorization
+
+The Bearer authorization is not linked to a credential managed in the Marketplace. This is the token the provider already gave you.
+
+When selecting this authorization type, you are presented a value box where you can paste your token.
+
+Once pasted, click **Authorize** button. The value is change to stars, and **Authorize** changes to **Logout**. If you logout, the bearer is erased and you must paste it again. If you click **Close**, the side panel is closed and you are now ready to [try a method](#try-a-method).
```
Comment (Ramoonaa) [inline content/en/docs/manage_marketplace/consumer_experience/api_tryoput.md:78]: value is changed

File: content/en/docs/manage_marketplace/consumer_experience/api_tryoput.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+First, you have to find the Product / APIs you want to test. Browse the product list and find the appropriate one. You can use the [search capabilities](/docs/manage_marketplace/consumer_experience/searching).
+
+Once you find the product, select the appropriate resources to open the *resources details* page. By default, the **Methods** tab is visible. This tab shows all the methods and models the API is using.
+
+To call an API, you will need to first select the credentials you want to use to access the API and then call the API itself.
+
+## Select your credentials
+
+To select the credentials for calling an API, you must either click **Authorize** to globally use credentials or **lock**, located beside the method, to use a credentials for that specific method.
```
Comment (Ramoonaa) [inline content/en/docs/manage_marketplace/consumer_experience/api_tryoput.md:21]: or lock icon

File: content/en/docs/manage_marketplace/consumer_experience/api_tryoput.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+Learn how to use the credentials for trying out APIs.
+
+## Where do I start?
+
+Once your credentials have been [created](/docs/manage_marketplace/consumer_experience/credential_management#create-credentials) and provisioned by the provider, ensure that these credentials work as expected.
+
+First, you have to find the Product / APIs you want to test. Browse the product list and find the appropriate one. You can use the [search capabilities](/docs/manage_marketplace/consumer_experience/searching).
+
+Once you find the product, select the appropriate resources to open the *resources details* page. By default, the **Methods** tab is visible. This tab shows all the methods and models the API is using.
```
Comment (Ramoonaa) [inline content/en/docs/manage_marketplace/consumer_experience/api_tryoput.md:15]: resource details page

**Review by Ramoonaa — APPROVED** [review]

**Pull request merged**